knimin/tests/test_ag_new_kit.py

```diff
@@ -56,9 +56,12 @@ def test_post(self):
         tag = 'abc'
 
         # check for correct results
+        projects = db.getProjectNames()
+        project1 = projects[0]
+        project2 = projects[1]
         response = self.post('/ag_new_kit/',
                              {'tag': tag,
-                              'projects': ['PROJECT2', 'PROJECT5'],
+                              'projects': [project1, project2],
```

Comment:
note that what I did is creating two variables to store the name of two projects. Before they're strings 😃